provisioning/service/tests/Config/TpmAttestationTests.cs

```diff
@@ -164,6 +178,27 @@ public void TpmAttestation_SucceedOnSerializationWithoutStorageRootKey()
             TestAssert.AreEqualJson(expectedJson, json);
         }
 
+        [TestMethod]
+        [TestCategory("DevService")]
+        public void TpmAttestation_SucceedOnSerializationWithEmptyStorageRootKey()
```

Comment:
Do we already have a test with a null/empty EK?

Reply:
`TpmAttestationTests` already has tests for null/empty EK. Since serialization can occur only after the TpmAttestation object has been created, null/empty EK would fail TpmAttestation object creation itself, thereby failing any subsequent actions such as serialization etc.